lib/semian/net_http.rb

```diff
@@ -36,6 +36,7 @@ def semian_identifier
       ::Timeout::Error, # includes ::Net::ReadTimeout and ::Net::OpenTimeout
       ::SocketError,
       ::Net::HTTPBadResponse,
+      ::Net::HTTPFatalError,
```

Comment:
> I think your option should ensure that ::Net::HTTPFatalError is in resource exceptions

Just to clarify, you think we should only have the `::Net::HTTPFatalError` in the list of exceptions if the option `fatal_server_errors` is enabled?